### ISSUE TITLE
Server.py and README instructions for model with db7f3d3 appear to be out of sync

### ISSUE THREAD

**Issue by nmstoker** [mozilla/TTS#154]
_No code to fix as such, although could clarify the instructions perhaps._

The instructions say to use the model that matches the commit but that appears to be inaccurate with regard to the model that goes with commit db7f3d3 as using server.py from that commit won't work with the model and you need to use it with the current server.py from master (as of now at least)

If you don't do this, you'll get the kind of problems seen here: https://github.com/mozilla/TTS/issues/150#issuecomment-482802000

I'm sure in time this will be superceded, but thought I'd record it here so people don't all run into the same problem.

**Comment by vitaly-zdanevich** [comment]
Thank you for help, I tried `best_model.pth.tar` with `config.json` for [db7f3d3](https://github.com/mozilla/TTS/commit/db7f3d36e7768f9179d42a8f19b88c2c736d87eb) downloaded from Google Drive on the `master` branch, I got this error:
```
/home/vitaly_zdanevich/.local/lib/python3.6/site-packages/numba/errors.py:105: UserWarning: Insufficiently recent colorama version found. Numba requires colorama >= 0.3.9
  warnings.warn(msg)
 > Loading model ...
 | > model config:  /home/vitaly_zdanevich/TTS/config.json
 | > model file:  /home/vitaly_zdanevich/TTS/best_model.pth.tar
 > Setting up Audio Processor...
 | > bits:None
 | > sample_rate:22050
 | > num_mels:80
 | > min_level_db:-100
 | > frame_shift_ms:12.5
 | > frame_length_ms:50
 | > ref_level_db:20
 | > num_freq:1025
 | > power:1.5
 | > preemphasis:0.98
 | > griffin_lim_iters:60
 | > signal_norm:True
 | > symmetric_norm:False
 | > mel_fmin:0
 | > mel_fmax:None
 | > max_norm:1.0
 | > clip_norm:True
 | > do_trim_silence:True
 | > n_fft:2048
 | > hop_length:275
 | > win_length:1102
Traceback (most recent call last):
  File "server/server.py", line 16, in <module>
    config.model_config, config.use_cuda)
  File "/home/vitaly_zdanevich/TTS/server/synthesizer.py", line 39, in load_model
    self.model.load_state_dict(cp['model'])
  File "/home/vitaly_zdanevich/.local/lib/python3.6/site-packages/torch/nn/modules/module.py", line 769, in load_state_dict
    self.__class__.__name__, "\n\t".join(error_msgs)))
RuntimeError: Error(s) in loading state_dict for Tacotron:
        size mismatch for embedding.weight: copying a param with shape torch.Size([61, 256]) from checkpoint, the shape in current model is torch.Size([129, 256]).
```

**Comment by erogol** [comment]
will be solved by #156

**Comment by erogol** [comment]
@vitaly-zdanevich @nmstoker  Consider to use the branch fix_db7f3d3
